### ISSUE TITLE
00 -- David White

### ISSUE THREAD

**Issue by s4dmw** [TIY-Durham/2015-FALL-FEE#12]
- [x] **GitHub-ing to Know You**
  - [x] [Create a GitHub account](https://github.com/signup)
  - [x] [Edit your GitHub profile](https://github.com/settings/profile) to provide:
  - [x] A picture, so we can recognize your face.
  - [x] Your full name, so we can recognize your name.
  - [x] A valid public email address, so we can contact you.
  - [x] Find and star [the class project](https://github.com/TIY-Durham/2015-FALL-FEE)
  - [x] Read [the Home page on the class wiki](https://github.com/TIY-Durham/2015-FALL-FEE/wiki)
- [x] **You've Got Issues**
  - [x] [Create a new _WIP Issue_](https://github.com/TIY-Durham/2015-FALL-FEE/issues/new)
  - [x] Name it `00 -- YOUR NAME` (where `YOUR NAME` is _your_ name)
  - [x] Make this checklist the description
  - [x] Start checking off boxes... :astonished:
- [x] **Send all my GitHub to you...**
  - [x] Learn all you can about `git` and GitHub!
    - Treehouse has [a Git Basics course](http://teamtreehouse.com/library/git-basics)
  - CodeSchool has [a `git` path](https://www.codeschool.com/paths/git)
    - [x] Complete: [Try Git](http://try.github.com)
  - [x] GitHub has a couple of Guides:
    - [x] [Hello World](https://guides.github.com/activities/hello-world/)
    - [x] [Forking Projects](https://guides.github.com/activities/forking/)
  - [x] Add a link to your `Hello-World` and `Fork-and-Spoon` repos as a comment on your _WIP Issue_
- [x] **Command Line Training**
  - [x] Read: http://eddywashere.com/blog/an-intro-to-the-os-x-command-line/
  - [x] Complete: https://www.codecademy.com/courses/learn-the-command-line/
  - [ ] **BEAST MODE** http://cli.learncodethehardway.org/book/
- [x] **INSTALL ALL THE THINGS!**
  - [x] [Install Homebrew](http://brew.sh) on your Mac
  - [x] Open `Terminal.app`; Don't Panic.
  - [x] [Install Homebrew and the `brew` command](http://brew.sh/#install)
  - [x] Read [the Homebrew Wiki](https://github.com/Homebrew/homebrew/tree/master/share/doc/homebrew#readme) to learn more about Homebrew
  - [x] Using the `brew` command, install the following packages:
    - [x] `zsh`
  - [x] `git`
  - [x] `hub`
  - [x] `node`
  - [x] Add the output of the following commands as comments on your _WIP Issue_:
    - [x] `which zsh && zsh --version`
    - [x] `which git && git --version`
    - [x] `which hub && hub --version`
    - [x] `which node && node --version`
  - [x] Install the following (bonus if you use `brew`):
    - [x] [Google Chrome](http://google.com/chrome)
    - [x] [Atom](http://atom.io)
    - [x] [Dash](http://kapeli.com/dash)
    - [x] [Slack](http://www.slack.com)
    - [x] [SourceTree](https://www.sourcetreeapp.com/)
- [x] **Prezto Change-oh!**
  - [x] Find and _fork_ the Prezto project on GitHub
  - [x] Clone _your fork_ of Prezto into your home directory
  - [x] Follow the installation instructions in the `README.md` file
- [x] **Take a Moment to Reflect**
  - [x] Answered question 1 in a comment on your WIP Issue.
  - [x] Answered question 2 in a comment on your WIP Issue.

**Comment by s4dmw** [comment]
https://github.com/s4dmw/hello-world

https://github.com/s4dmw/Spoon-Knife

**Comment by s4dmw** [comment]
/usr/local/bin/zsh
zsh 5.0.8 (x86_64-apple-darwin14.4.0)

**Comment by s4dmw** [comment]
/usr/local/bin/git
git version 2.3.2 (Apple Git-55)

**Comment by s4dmw** [comment]
/usr/local/bin/hub
git version 2.5.0
hub version 2.2.1

**Comment by s4dmw** [comment]
/usr/local/bin/node
v0.12.7

**Comment by s4dmw** [comment]
Question 1:

I think the biggest difficulty for me will be the time commitment, especially if i a having trouble grasping a concept. While i understand/expect this course will dominate my life for the next 12 weeks, being able to have a little fun will help me keep my sanity and make it through the course.

In the past, i have found that when a task becomes overwhelming, taking a break from it is immensely helpful for me (even if it's just an hour or two of doing something different). I always feel refreshed afterwards and can often quickly figure out what i need to differently to get the task finished.

**Comment by s4dmw** [comment]
Question 2:

I enjoyed learning the basics of html and CSS, and brushing up on my command line skills.

Other than buying a macbook, some of the more involved (i realize i haven't seen anything yet) loops in/nested loops in js were a bit confusing. I learned that by picking them apart to debug portions of the loops, i could usually get them working correctly. Most of the developer tools are new to me as well but i think i will pick them up pretty quickly with use/practice.
